crates/core/tedge_mapper/src/mapper.rs

```diff
@@ -70,6 +70,11 @@ impl Mapper {
 
     #[instrument(skip(self), name = "messages")]
     async fn subscribe_messages(&mut self) -> Result<(), MqttClientError> {
+        let init_messages = self.converter.init_messages();
+        for init_message in init_messages.into_iter() {
+            self.client.publish(init_message).await?
+        }
```

Comment:
Will these init messages be read from the operation configuration file?

Reply:
Not the messages, but the content of them (it) will be read from operations directory.